### DISCUSSION TITLE
Open hi-res in a new tab

### DISCUSSION THREAD

**Discussion by tsvetkovv** [photoprism/photoprism#2604] [Ideas]
Hi, thank you for putting tremendous effort into this!

If I open the photo on small sized-display, there is no way to zoom in and get a hi-res thumbnail or original. Except by downloading a file.
It would be nice to have a function to open (not download) an original file (if it's jpg) or hi-res thumbnail. Another way is to fetch hi-res thumbnail if we use zoom.
What do you think?

**Comment by Sanookmakmak** [comment]
I would like to combine my IIPImage zoomserver for gigapixel images with PhotoPrism.

https://iipimage.sourceforge.io/

**Comment by graciousgrey** [comment]
Zooming will be improved once we have built a new photo viewer --> https://github.com/photoprism/photoprism/issues/1307.
